L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp2.cc

```diff
@@ -628,8 +754,10 @@ int HGCalTriggerGeometryV9Imp2::detIdWaferType(unsigned det, unsigned layer, sho
 }
 
 unsigned HGCalTriggerGeometryV9Imp2::layerWithOffset(unsigned id) const {
+  // not used for HFNose
```

Comment:
This function should be able to handle HFNose, otherwise it would always return 0 in that case?

Reply:
implemented